app/modules/collaborations/models.py

```diff
@@ -63,6 +63,15 @@ class CollaborationUserAssociations(db.Model, HoustonModel):
     def delete(self):
         self.collaboration.user_deleted(self)
 
+    def has_read(self):
```

Comment:
Added to avoid user code having to delve into the collaboration to find this info